test_s3_tagger.py

```diff
@@ -464,43 +507,6 @@ def test_tag_folder(csv_data):
     assert response5["TagSet"][2]["Value"] == "true", "Object was not tagged correctly"
 
 
-@mock_s3
-def test_items_that_dont_match_expected_pattern():
```

Comment:
Why have we removed this test?

Reply:
Duplicate / moved up and renamed. See: `test_tag_object_unrecognised_key_for_database_and_table_name_length_two`